HW4_Mukhametshina/amino_acid_tools.py

```diff
@@ -0,0 +1,241 @@
+amino_acid = 'ARNDCEQGHILKMFPSTWYVUOarndceqghilkmfpstwyvuo'
```

Comment:
1) У тебя здесь набор аминокислот, поэтому лучше называть переменную во множественном числе
2) Такие штуки которые мы выносим в начало кода называются "константами" и в питоне их принято называть капсом. 
То есть в идеале это было бы так:
```suggestion
AMIINO_ACIDS = 'ARNDCEQGHILKMFPSTWYVUOarndceqghilkmfpstwyvuo'
```
Но в целом правильно сделала, просто пару советов по доработке